slack-archive/src/test/java/juja/microservices/slack/integration/ArchiveServiceIntegrationTest.java

```diff
@@ -26,7 +29,15 @@
     private ArchiveServiceImpl service;
 
     @Test
-    public void saveMessageTest() {
+    @UsingDataSet(locations = "/datasets/channels.json")
+    public void getChannelsTest() {
+        ChannelDTO channelOne = new ChannelDTO("CHANIDONE", "flood");
+        ChannelDTO channelTwo = new ChannelDTO("CHANIDTWO", "feedback");
+
+        List<ChannelDTO> actual = service.getChannels();
 
+        assertNotNull(actual);
```

Comment:
If dataset contain more channels besides this two - test does'n fail.
Maybe   assertThat(actual, IsIterableContainingInAnyOrder.containsInAnyOrder(expected.toArray())); 
instead of three Asserts?

Reply:
Good point, thanks! I fixed it.